src/query/query_hash_scan.c

```diff
@@ -1387,16 +1388,13 @@ fhs_initialize_bucket_new_page (THREAD_ENTRY * thread_p, PAGE_PTR page_p, void *
 static int
 fhs_initialize_dk_bucket_new_page (THREAD_ENTRY * thread_p, PAGE_PTR page_p, void *args)
 {
-  VPID next_bucket;
+  VPID null_vpid = { NULL_VOLID, NULL_PAGEID };
```

Comment:
fyi.
The definition "VPID_INITIALIZER" is available.
VPID null_vpid = VPID_INITIALIZER;

Reply:
I fixed it.